platforms/stellar-disbursement-platform/admin-guide/user-interface/70-circle-configutration.mdx

```diff
@@ -1,25 +1,66 @@
 ---
 title: Circle Configuration
-id: circle-configutration
+id: circle-configuration
 sidebar_position: 70
 ---
 
+## Using a Shared Circle Account Across All Tenants
```

Comment:
Just to clarify, every tenant has to configure their own Circle account. You can have all tenants configuring and using the same account, but that configuration still has to be repeated for every tenant and there is no way to share it across tenants automatically .

Reply:
ah yea ill make adjustments so this is clear. thank you